docs/software/rpm-development-guide.md

```diff
@@ -159,14 +164,20 @@ without the prefix component, followed by the sha1sum of the file:
 
 > `<PACKAGE>/<VERSION>/<FILE> sha1sum=<SHA1SUM>`
 
+Obtain the sha1sum by running the `sha1sum` command with the source file as an argument, i.e.
+```console
+$ sha1sum /p/vdt/public/html/upstream/<PACKAGE>/<VERSION>/<FILE>
```

Comment:
You are giving the instructions for this twice?

Reply:
Yeah, I feel like it's kinda hidden so I want to make sure people get it.

Reply:
Arright, well, for the record i feel like it's overkill, as well as giving the full path to the file in the example, but whatever you like.